checker/src/org/checkerframework/checker/nullness/NullnessAnnotatedTypeFactory.java

```diff
@@ -507,8 +507,26 @@ public NullnessTypeAnnotator(InitializationAnnotatedTypeFactory<?, ?, ?, ?> atyp
 
     @Override
     public AnnotationMirror getFieldInvariantAnnotation() {
-        Elements elements = processingEnv.getElementUtils();
-        return AnnotationUtils.fromClass(elements, NonNull.class);
+        return NONNULL;
+    }
+
+    /**
+     * Returns whether or not field has the invariant annotation.
```

Comment:
Same comments as on javadoc for the overridden method. Can the common parts be inherited to reduce duplication?